pkg/apiserver/proxy.go

```diff
@@ -151,6 +151,8 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 	httpCode = http.StatusOK
 	newReq.Header = req.Header
+	newReq.ContentLength = req.ContentLength
+	newReq.TransferEncoding = req.TransferEncoding
```

Comment:
I'm pretty sure we should only be setting ContentLength. TransferEncoding is interpreted, and the request body wrapped in an interpreting reader before we get it (that's why a request sent as "chunked" comes in with no transfer encoding, and reading the body returns unchunked data).